src/components/Notification/Notification.stories.mdx

```diff
@@ -118,17 +118,44 @@ be rendered inside other landmark elements such as the `header` and `footer`
 landmark elements. Adding a `Notification` component inside an HTML `main` landmark
 element is acceptable.
 
+### Arial Label
```

Comment:
Small typo: should be "ARIA".

Reply:
Ha!  We watched `The Little Mermaid` this past weekend.  It must have been on my mind.  I'll fix that.